crawl-ref/source/art-data.txt

```diff
@@ -547,6 +547,7 @@ TILE:    urand_piercer
 TILE_EQ: great_bow
 BRAND:   SPWPN_PENETRATION
 EV:      -2
+BOOL:    nogen
```

Comment:
The start of this entry should have `# TAG_MAJOR_VERSION == 34` to tag Piercer for removal.